lib/error-matcher.js

```diff
@@ -88,7 +88,7 @@ module.exports = (function () {
 
     var output = '';
     var lastEnd = 0;
-    for (var matchIndex in this.currentMatch) {
+    for (var matchIndex = 0; this.currentMatch.length && matchIndex <= this.currentMatch.length; matchIndex++) {
```

Comment:
This doesn't seem right. The index could never be equal to the length of the array.